src/components/SlateEditor/PlainSlateEditor.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Editor } from 'slate';
+
+const PlainSlateEditor = ({ onChange, value, ...rest }) =>
```

Comment:
Forsåvidt enig i denne. Var mere et navn for å skille mellom slate og draftjs til vi har fjernet draft.